metpy/plots/tests/test_skewt.py

```diff
@@ -77,3 +77,7 @@ def test_hodograph_units():
     hodo.plot(u, v)
     hodo.plot_colormapped(u, v, np.sqrt(u * u + v * v), cmap='Greys')
     return fig
+
```

Comment:
Need two blank lines between global-level functions--it's a [pep8](https://www.python.org/dev/peps/pep-0008/) thing.

This was caught by flake8 on Travis:
```
/home/travis/build/Unidata/MetPy/metpy/plots/tests/test_skewt.py:81:1: E302 expected 2 blank lines, found 1
```